src/thunderbots/software/ai/hl/stp/tactic/move.cpp

```diff
@@ -0,0 +1,15 @@
+#include "ai/hl/stp/tactic/move.h"
+
+MoveTactic::MoveTactic(const Point &destination) : destination(destination)
+{
+}
+
+std::pair<Robot, Team> MoveTactic::selectRobot(const Team &available_robots)
+{
+    return std::make_pair(available_robots.getAllRobots().at(0), available_robots);
+}
+
+std::unique_ptr<Intent> MoveTactic::getNextIntent(const World &world, const Robot &robot)
+{
+    return std::make_unique<MoveIntent>(robot.id(), destination, Angle::zero(), 0.0);
+}
```

Comment:
Is this the completed implementation for the `move` tactic? In general, this PR should only be for interface stuff right (and not for concrete implementations)?

Reply:
Needed a placeholder for the node to run completely.

Reply:
This PR is for interfaces, but when running the node I cannot return an "empty" pointer of the Interface type. I do need to be able to call a concrete instance for the code to run. I'm just adding as little as possible to allow the nodes to still run. I've made it a bit more clear in my next changes that the implementation is a placeholder.